bin/makepart

```diff
@@ -89,6 +89,7 @@ sed 's/#.*//;/^[[:blank:]]*$/d' \
 		fi
 		column -t "$fstab" > "$rootfs_work/etc/fstab"
 		column -t "$veritytab" > "$rootfs_work/etc/veritytab"
+		chcon system_u:object_r:etc_t:s0 "$rootfs_work/etc/veritytab"
```

Comment:
While this works, I don't think this is a proper solution. It does not use the file table with default contexts, which might even be changed by other changes later.

Maybe the following will work?

* Create an empty `/etc/veritytab` in the proper `file.include`.
* Just append new entries in `makepart`.